src/main/java/org/cbioportal/service/impl/ClinicalDataDensityPlotServiceImpl.java

```diff
@@ -18,8 +19,14 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
+
 @Service
 public class ClinicalDataDensityPlotServiceImpl implements ClinicalDataDensityPlotService {
+
+    @Cacheable(
```

Comment:
@haynescd ultimately i want to make the condition "unfiltered" but if you look up the way singleStudyUnfiltered is implemented, it has to be calculated on every method call. Would like to see if there's a more general way.  But for now, we can just put everything in cache.  It's ok.  It won't overwhelm the cache on genie as there's not that much traffic.  Can explain more later.
```
    @Cacheable(
               cacheResolver = "staticRepositoryCacheOneResolver",
               condition = "@cacheEnabledConfig.getEnabled() && #singleStudyUnfiltered" 
    )
```